8Knot/pages/index/index_layout.py

```diff
@@ -90,6 +90,7 @@
                             dbc.Nav(
                                 [
                                     dbc.NavLink("Welcome", href="/", active="exact"),
+                                    dbc.NavLink("Repo Info", href="/repo_info", active="exact"),
```

Comment:
Maybe this is a good opportunity to use the term `Repo Overview` because we already use `Info` in a tab.

Reply:
Good call

Reply:
++